src/librustdoc/html/render.rs

```diff
@@ -4063,11 +4063,25 @@ fn get_methods(
         .filter_map(|item| match item.name {
             Some(ref name) if !name.is_empty() && item.is_method() => {
                 if !for_deref || should_render_item(item, deref_mut) {
-                    Some(format!(
-                        "<a href=\"#{}\">{}</a>",
-                        get_next_url(used_links, format!("method.{}", name)),
-                        name
-                    ))
+                    let mut emojis = String::new();
+                    if let clean::FunctionItem(ref func) | clean::ForeignFunctionItem(ref func) = item.inner {
+                        if func.header.unsafety == hir::Unsafety::Unsafe {
+                            emojis += "⚠";
+                        }
+                    }
+                    if let Some(stab) = item.stability.as_ref().filter(|stab| stab.level == stability::Unstable) {
+                        let is_rustc_private = stab.feature.as_ref().map(|s| &**s) == Some("rustc_private");
```

Comment:
```suggestion
                        let is_rustc_private = stab.feature.as_deref() == Some("rustc_private");
```

Reply:
Nice, should I change this for https://github.com/rust-lang/rust/blob/810fa1e812168b68e871bb194f77e5e4394a1b05/src/librustdoc/html/render.rs#L2155 on another commit?

Reply:
I would change it too.